exe/cli/cmd/ffs_getdefaultcidconfig.go

```diff
@@ -0,0 +1,51 @@
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"time"
+
+	"github.com/caarlos0/spin"
+	"github.com/ipfs/go-cid"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func init() {
+	ffsGetDefaultCidConfigCmd.Flags().StringP("token", "t", "", "FFS auth token")
+
+	ffsCmd.AddCommand(ffsGetDefaultCidConfigCmd)
+}
+
+var ffsGetDefaultCidConfigCmd = &cobra.Command{
+	Use:   "getDefaultCidConfig [cid]",
+	Short: "Retuns the default config prepped for the provided cid",
+	Long:  `Retuns the default config prepped for the provided cid`,
```

Comment:
Typo `prepped`

Reply:
I think that's the right way to spell it, no?

Reply:
Sorry, you're right. I looked in the dictionary. The past tense of `prep`.